src/movingaverages.jl

```diff
@@ -110,6 +130,15 @@ function ema(a::Array{T,N}, n::Int; wilder=false) where {T,N}
     vals[n:end, :]
 end
 
+function env(a::Array{T,N}, n::Int; e::Float64 = 0.1) where {T,N}
+    s = sma(a, n)
+
+    upper = s + s * e
+    lower = s - s * e
+
+    upper, lower
```

Comment:
How about merging this also via `[upper lower]` ?

Reply:
I think `[lower upper]` makes more sense.

Reply:
oh, okay.